### PR TITLE
feat(3200): V2 metrics page redirect to V1

### PR DESCRIPTION
**Pull request by VonnyJap**
## Context

The current V2 metrics page is showing empty metrics page content. 

Screenshot 2024-11-18 at 7 31 12 AM

## Objective

Redirect the content to V1 until the new component is created

## References

https://github.com/screwdriver-cd/screwdriver/issues/3200

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/v2/pipeline/metrics/route.js [hunk 1]
```diff
@@ -1,3 +1,13 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class NewPipelineMetricsRoute extends Route {}
+export default class NewPipelineMetricsRoute extends Route {
+  @service router; // Ensure router service is injected
```
Comment (sagar1312) [inline app/v2/pipeline/metrics/route.js:5]: Code seems to be self explanatory. We can remove the comments.
```suggestion
  @service router;
```

File: app/v2/pipeline/metrics/route.js [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+  beforeModel(transition) {
+    const id = transition.to.parent.params.pipeline_id;
+    // Use RouterService to handle redirection
```
Comment (sagar1312) [inline app/v2/pipeline/metrics/route.js:9]: Same here
```suggestion
```

**Review by minghay — CHANGES_REQUESTED** [review]
Just change the route in the nav component of the V2 UI instead of doing all of the redirects.

**Review by minghay — COMMENTED** [review]
You accidentally committed changes to the `app/v2/route.js`

**Review by minghay — APPROVED** [review]

**Pull request merged**